.github/workflows/ci.yml

```
@@ -0,0 +1,106 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      # Feature branches beginning with "feature."
+      - feature.*
+  pull_request:
```

```suggestion
  pull_request: {branches: ['**']}
```

Otherwise I don't think this will actually run.

---

it will run. When you don't define `branches` or `tags`, it runs on all references

---

That wasn't my experience with https://github.com/sass/sass/pull/2992... it didn't run until I pushed https://github.com/sass/sass/pull/2992/commits/3386ec27dcf3cb726ae3984dfdfdb70d8a75b2a0.

---

then the github doc would be wrong

---

also, I have other repos where this works.

---

Looks like you're right, https://github.com/sass/sass/pull/2998 is running actions now. No idea why my previous attempt didn't work.